test/utils/computeLedgerHash/index.ts

```diff
@@ -1,35 +1,27 @@
 import assert from 'assert-diff'
```

Comment:
I wouldn't put this in an `index` file - it's much neater if it's just `test/utils/computeLedgerHash.ts`. Same with all the other utils files.